storage/sql_hooks.go

```diff
@@ -0,0 +1,93 @@
+package storage
+
+import (
+	"context"
+	"database/sql"
+	sql_driver "database/sql/driver"
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/gchaincl/sqlhooks"
+	pq "github.com/lib/pq"
+	sqlite3 "github.com/mattn/go-sqlite3"
+)
+
+type sqlHooks struct {
+	SQLQueriesLogger *log.Logger
+}
+
+type sqlHooksKey int
+
+const (
+	sqlHooksKeyQueryBeginTime sqlHooksKey = iota
+)
+
+// LogFormatterString is format string for sql queries logging
+// first arg is query string
+// second arg is params array
+const logFormatterString = "query `%+v` with params `%+v`"
+
+func (h *sqlHooks) Before(ctx context.Context, query string, args ...interface{}) (context.Context, error) {
```

Comment:
dtto - linter won't be happy about missing docstrings ;)

Reply:
fixed